src/components/Table/GridTable.tsx

```diff
@@ -437,7 +437,7 @@ function renderDiv<R extends Kinded>(
       <div
         css={{
           ...(style.betweenRowsCss ? Css.addIn(`& > div > *`, style.betweenRowsCss).$ : {}),
-          ...(style.firstNonHeaderRowCss ? Css.addIn(`& > div:first-of-type > *`, style.firstNonHeaderRowCss).$ : {}),
```

Comment:
Just curious,  it looks like we used to apply this to the cells, and now we're applying it directly to the row? That seems better, just following along.

Reply:
Yup, thats true. This is a relic of the `display: contents` days when we couldn't apply styles directly to rows.